### PR TITLE
Hcg 44

### PR DESCRIPTION
**Pull request by philbrookes**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -47,6 +49,7 @@ kcp:
 	cp ./tmp/kcp/bin/shard-proxy $(shell pwd)/bin
 	cp ./tmp/kcp/bin/syncer $(shell pwd)/bin
 	cp ./tmp/kcp/bin/virtual-workspaces $(shell pwd)/bin
+	cp ./tmp/kcp/bin/ingress-controller $(shell pwd)/bin
```
Comment (mikenairn) [inline Makefile:52]: This duplicates the name of the controller we are building, do we not need to rename ours?

Comment (mikenairn) [reply Makefile:52]: Alternative to renaming ours, would be to stop copying all these into our local .bin dir. Do we even need to do this? Is it not possible to tell kcp to start these controllers?

File: utils/local-setup.sh [hunk 2]
```diff
@@ -129,6 +132,20 @@ fi
 echo "Exporting KUBECONFIG=.kcp/admin.kubeconfig"
 export KUBECONFIG=.kcp/admin.kubeconfig
 
+echo ""
+echo "Starting Ingress Controller"
+"${INGRESS_CONTROLLER_BIN}" --kubeconfig="${KUBECONFIG}" --envoy-listener-port=8181 --envoy-xds-port=18000 &> ${INGRESS_CONTROLLER_LOG_FILE} &
```
Comment (mikenairn) [inline utils/local-setup.sh:137]: Adding the envoy listener appears to stop it doing the aggregateLeaveStatus https://github.com/kcp-dev/kcp/blob/main/pkg/reconciler/ingresssplitter/ingress.go#L51 https://github.com/kcp-dev/kcp/blob/main/cmd/ingress-controller/main.go#L91-L92. Starting it like this makes it work as it did before:

```
"${INGRESS_CONTROLLER_BIN}" --kubeconfig="${KUBECONFIG}" --envoy-listener-port=0 --envoy-xds-port=0 &> ${INGRESS_CONTROLLER_LOG_FILE} &
```

**Comment by mikenairn** [comment]
@maleck13 Can you confirm we are holding off on this change for now?

**Comment by maleck13** [comment]
Yes we can leave this for now

**Pull request closed without merging**